[DAL] CSV Data Source specification (#199)
oneapi-src/oneAPI-spec: source/elements/oneDAL/source/glossary.rst

        :capterm:`responses <Response>`. The goal is to find hidden patters inside
        :capterm:`feature vectors <Feature vector>` and dependencies between them.

    CSV file

>         A comma-separated values file (csv) is a text file. Each line of the csv file is a record, containing fields separated by the delimiters.

Reviewer: ```suggestion
        A comma-separated values file (csv) is a type of a text file. Each line in a CSV file is a record containing fields that are separated by the delimiter.
```

>         Fields can have numerical or text format, where text usually refers to some categorized values.

Reviewer: categorical, not categorized

Author: Done.

>         Fields can have numerical or text format, where text usually refers to some categorical values.

Reviewer: ```suggestion
        Fields can be of a numerical or a text format. Text usually refers to categorical values.
```

>         Delimiter is a comma by default, but generally can be any given symbol.

Reviewer: ```suggestion
        By defaul, the delimiter is a comma, but, generally, it can be any character.
```
